boa3/compiler/codegenerator/codegenerator.py

```diff
@@ -891,8 +891,12 @@ def convert_literal(self, value: Any) -> int:
             self.insert_none()
         elif isinstance(value, (bytes, bytearray)):
             self.convert_byte_array(value)
+        elif isinstance(value, Sequence):
+            self.convert_sequence_literal(value)
+        elif isinstance(value, dict):
+            self.convert_dict_literal(value)
```

Comment:
It would be very interesting if we could replace this long `isInstance` if-else chain with a single call of `convert_literal` of an interface. I say this because it looks like that maybe a new type could appear and will need a new implementation, and perhaps the developer will forget to do it.

Let me know if you have troubles to implement it, it is impossible to implement on this situation or you disagree with me.